druid/src/env.rs

```diff
@@ -167,14 +167,15 @@ impl Env {
     /// Set by the `debug_paint_layout()` method on [`WidgetExt`]'.
     ///
     /// [`WidgetExt`]: trait.WidgetExt.html
-    pub(crate) const DEBUG_PAINT: Key<bool> = Key::new("druid.built-in.debug-paint");
+    pub(crate) const DEBUG_PAINT: Key<bool> = Key::new("org.linebender.druid.theme.debug-paint");
```

Comment:
for these things that aren't actually part of the _theme_ per se I would do `org.linebender.druid.built-in.XXX`